lib/components/search/SearchOption.js

```diff
@@ -60,7 +60,7 @@ const SearchOption = ({
         <div className={classes.proportionContainer}>
           <div
             className={classes.proportion}
-            style={{ width: `${Math.round(proportion * 100)}%` }}
+            style={{ width: `${proportion * 100}%` }}
```

Comment:
You could also move these inline styles to the definition of the styles object since this component is already being wrapped with the `withStyles` HOC.

Reply:
Nevermind, these styles depend on a prop.